app/build.gradle

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Vandolf Estrellado
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion '25.0.2'
+
+    defaultConfig {
+        applicationId "com.vestrel00.daggerbutterknifemvp"
+        versionCode 1
+        versionName '1.0.0'
+
+        minSdkVersion 21
```

Comment:
This has been updated to 17. See https://github.com/vestrel00/android-dagger-butterknife-mvp/commit/89dcdfaec7cd6b78fa4eeab2383065e567599486